library/std/src/os/darwin/mod.rs

```diff
@@ -0,0 +1,17 @@
+//! Platform-specific extensions to `std` for Darwin / Apple platforms.
+//!
+//! This is available on the following operating systems:
+//! - macOS
+//! - iOS
+//! - tvOS
+//! - watchOS
+//! - visionOS
+//!
+//! Note: This module is called "Darwin" as that's the name of the underlying
+//! core OS of the above operating systems, but it should not be confused with
+//! the `-darwin` suffix in the `x86_64-apple-darwin` and
+//! `aarch64-apple-darwin` target names, which are mostly named that way for
+//! legacy reasons.
+
+pub(crate) mod fs;
+pub(crate) mod raw;
```

Comment:
Can these be `pub(super)` instead? And I'd like a comment on `raw` since it's only for reexport on some systems, even if we make this module public, like...
```suggestion
pub(super) mod fs;
/// deprecated, only for public reexport on macOS/iOS because their platform support precedes the decision to remove these.
pub(super) mod raw;
```

Reply:
I think possibly only `raw` can be `pub(super)` due to certain tests? idk.

Reply:
I've written it as a slightly different comment that also talks about the usage in `std::os::unix::raw`